apps/gyrokinetic.c

```diff
@@ -1132,10 +1132,12 @@ gkyl_gyrokinetic_app_write_mom(gkyl_gyrokinetic_app* app, double tm, int frame)
       snprintf(fileNm, sizeof fileNm, fmt, app->name, app->species[i].info.name,
         app->species[i].info.diag_moments[m], frame);
 
-      // Rescale moment by inverse of Jacobian
-      gkyl_dg_div_op_range(app->species[i].moms[m].mem_geo, app->confBasis, 
-        0, app->species[i].moms[m].marr, 0, app->species[i].moms[m].marr, 0, 
-        app->gk_geom->jacobgeo, &app->local);      
+      // Rescale moment by inverse of Jacobian if not already re-scaled 
+      if (!app->species[i].moms[m].is_bi_maxwellian_moms && !app->species[i].moms[m].is_maxwellian_moms) {
```

Comment:
@JunoRavin same here, I would rename `bi_maxwellian` to `bimaxwellian`.

Also, should we just make the regular moments always be rescaled by 1/jacobgeo? They are used for LBO prim moms and for reactions. For LBO I think it doesn't matter whether we use J*moms or just moms. For reactions, since we use them to compute cross sections and Maxwellians, can't we just do all that with moms (instead of J*moms)?